src/app/freelancer/project/[project_id]/page.tsx

```diff
@@ -48,28 +53,64 @@ interface Project {
   updatedAt?: Date;
 }
 
+const getStatusBadge = (status: string | undefined) => {
+  switch (status?.toLowerCase()) {
+    case 'active':
+      return { text: 'ACTIVE', className: 'bg-blue-500 hover:bg-blue-600' };
+    case 'pending':
+      return { text: 'PENDING', className: 'bg-warning hover:bg-warning' };
+    case 'completed':
+      return { text: 'ACCEPTED', className: 'bg-success hover:bg-success' };
+    case 'rejected':
+      return { text: 'REJECTED', className: 'bg-red-500 hover:bg-red-600' };
+    default:
+      return { text: 'UNKNOWN', className: 'bg-gray-500 hover:bg-gray-600' };
+  }
+};
 export default function Dashboard() {
   const { project_id } = useParams<{ project_id: string }>();
   const [project, setProject] = useState<Project | null>(null);
+  const user = useSelector((state: RootState) => state.user);
+  const [bids, setBids] = useState([
+    {
+      userName: '',
+      current_price: '',
+      bid_status: '',
+      bidder_id: '',
+    },
+  ]);
+  const [exist, setExist] = useState(false);
 
   useEffect(() => {
     const fetchData = async () => {
       try {
         const response = await axiosInstance.get(
-          `/business/${project_id}/project`,
+          `/business/${project_id}/${user.uid}/project`,
         );
-        setProject(response.data.data);
+        setProject(response.data.data.data);
```

Comment:
Please check the response you get from api before assessment to a state variable.
Don't believe you will always get exact data.data.data. if response is slightly different your app will crash.
Use null check to make sure it doesn't.
Eg: response?.data?.data?.data --> this ensures that you will get undefined if correct hierarchy of data is not returned from api 
Then only set the value in a state variable if it is not undefined